backend/tests/requests.go

```diff
@@ -0,0 +1,153 @@
+package tests
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"shodo/models"
+)
+
+func (s *APITestSuite) sendRequest(method, url string, body []byte, authToken *string) (*http.Response, error) {
+	requestReader := bytes.NewReader(body)
+
+	req, err := http.NewRequest(method, baseUrl+url, requestReader)
+	if err != nil {
+		return nil, err
+	}
+
+	req.Header.Set(ContentType, ApplicationJSON)
+	if authToken != nil {
+		req.Header.Set(Authorization, *authToken)
+	}
+
+	client := &http.Client{}
+	return client.Do(req)
+}
+
+func (s *APITestSuite) sendJSONRequest(method, url string, request interface{}, authToken *string) (*http.Response, error) {
+	requestBytes, err := json.Marshal(request)
+	if err != nil {
+		return nil, err
+	}
+
+	return s.sendRequest(method, url, requestBytes, authToken)
+}
+
+func (s *APITestSuite) sendRegisterRequest(request models.RegisterUserRequest) (*http.Response, error) {
```

Comment:
why not use `httptest.ResponseRecorder`?

Reply:
Unfortunately I didn't understood your thought. Can you explain, please, how exactly should I use `httptest.ResponseRecorder`?

Reply:
Oh I see, your tests directory is basically e2e tests :relaxed: 
Ignore me here